frontend/src/components/Dapp.tsx

```diff
@@ -91,6 +91,35 @@ export class Contracts extends React.Component<Props, State> {
   }
 
   render(): React.ReactNode {
+    const DEFAULT_DECIMALS_SHOWN = 4;
+
+    const formatBalance = (
+      balance: BigNumber,
+      symbol: string,
+      decimalsShown: number = DEFAULT_DECIMALS_SHOWN,
+    ): string => {
+      const formatNumber = (): string => {
+        const value = balance.div(BigNumber.from(10).pow(this.props.sproutTokenData.decimals)).toString();
+
+        if (decimalsShown === 0) {
+          return `${value}`;
+        }
+
+        const string = balance.toString();
```

Comment:
I would prefer a more meaningful name than string for this, maybe this one can be named `balance` and the method param could be `balanceBN` or whatever you find better but not just `string`.